### ISSUE TITLE
koji_promote: store CG build ID as label

### ISSUE THREAD

**Issue by twaugh** [projectatomic/atomic-reactor#439]
Users will need to know the ID of the build object created using the Content Generator API.

**Comment by twaugh** [comment]
CC @maxamillion

**Comment by twaugh** [comment]
Seems like the CGImport() call will return a buildinfo map, like get_build() does:
  https://pagure.io/koji/pull-request/71
  https://pagure.io/koji/blob/master/f/hub/kojihub.py#_3278

So we can get the build ID like this:

```
buildinfo = session.CGImport(...)
build_id = buildinfo['id']
```

**Comment by twaugh** [comment]
Closed via #469.
